tensorflow_datasets/core/features/feature.py

```diff
@@ -444,12 +445,34 @@ def decode_ragged_example(self, tfexample_data):
     """
     return tf.ragged.map_flat_values(self.decode_batch_example, tfexample_data)
 
+  def _build_thumbnail_with_bbox(self, ex: np.ndarray) -> str:
+    """Returns the HTML str representation of an Image with BBoxes."""
+    PIL_Image = lazy_imports_lib.lazy_imports.PIL_Image
+    PIL_ImageDraw = lazy_imports_lib.lazy_imports.PIL_ImageDraw
+
+    SIZE = 150
+    blank_img = PIL_Image.new('RGB', (SIZE, SIZE), (255, 255, 255))
+    draw = PIL_ImageDraw.Draw(blank_img)
+
+    for i in range(ex.shape[0]):
+      # Rescale coordinates to match size of blank_image
+      ymin, xmin, ymax, xmax = ex[i, :]*SIZE
+      # Generate random rgb values for Bbox ouline
+      r, g, b = list(np.random.randint(0, 256, size=3))
+      draw.rectangle(((xmin, ymin), (xmax, ymax)), outline=(r, g, b))
+
+    img_str = utils.get_base64(lambda buff: blank_img.save(buff, format='PNG'))
+
+    return f'<img src="data:image/png;base64,{img_str}" alt="Img" />'
+
   def repr_html(self, ex: np.ndarray) -> str:
     """Returns the HTML str representation of the object."""
     return _repr_html(ex)
 
   def repr_html_batch(self, ex: np.ndarray) -> str:
     """Returns the HTML str representation of the object (Sequence)."""
+    if len(ex.shape) > 1 and ex.shape[1] == 4:
```

Comment:
This should be implemented in https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/core/features/bounding_boxes.py instead.

Similarly to `Image.repr_html`:

https://github.com/tensorflow/datasets/blob/26587ec6d4b23a501210a687be667e2f8256f32a/tensorflow_datasets/core/features/image_feature.py#L164